### PR TITLE
feat: ssr-catalogue add version info to about page

### PR DESCRIPTION
**Pull request by connoratrug**

### CODE REVIEW

File: apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/[catalogue]/about.vue [hunk 1]
```diff
@@ -277,11 +279,46 @@
...
+    key: `manifest`,
+    baseURL: useRuntimeConfig().public.apiBase,
+    method: "POST",
+    body: {
+      query: ` 
+    {
+      _manifest {
+        ImplementationVersion
+        SpecificationVersion
+        DatabaseVersion
```
Comment (mswertz) [inline apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/[catalogue]/about.vue:312]: This is nice

I supposed you also would like the catalogue version, i.e. the _schema.tables(name=Version).description?

Comment (connoratrug) [reply apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/[catalogue]/about.vue:312]: yes , thats is what we needed , but then i thought lets add what we have for now

File: apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/[catalogue]/about.vue [hunk 2]
```diff
@@ -277,11 +279,46 @@
             >
           </p>
         </ContentBlock>
+
+        <ContentBlock id="version" title="Version">
```
[the remark this answers is not in the document]
Comment (connoratrug) [reply apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/[catalogue]/about.vue:283]: yes , wasn't sure if we want component combining data fetch and render , or separate fetch -> store to application state -> render where needed. so just added to to a page

[the remark this answers is not in the document]
Comment (mswertz) [reply apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/[catalogue]/about.vue:283]: agreed in this case

File: apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/[catalogue]/about.vue (hunk 1 above)
Comment (mswertz) [reply apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/[catalogue]/about.vue:312]: okay, I approve this one first and this other thing can be next PR

**Review by mswertz — APPROVED** [review]

**Pull request merged**